### PR TITLE
chore: remove genesis hash from node configuration

### PR DESCRIPTION
**Pull request by fl0rek**

### CODE REVIEW

File: node/src/network.rs [hunk 1]
```diff
@@ -49,21 +48,6 @@ pub fn network_id(network: Network) -> &'static str {
...
-/// Get the hash of a genesis block for the given network.
-pub fn network_genesis(network: Network) -> Option<Hash> {
-    let hex = match network {
-        Network::Mainnet => "6BE39EFD10BA412A9DB5288488303F5DD32CF386707A5BEF33617F4C43301872",
-        Network::Arabica => "5904E55478BA4B3002EE885621E007A2A6A2399662841912219AECD5D5CBE393",
-        Network::Mocha => "B93BBE20A0FBFDF955811B6420F8433904664D45DB4BF51022BE4200C1A1680D",
-        Network::Private => return None,
-    };
-
-    let bytes = hex::decode(hex).expect("failed decoding genesis hash");
```
Comment (oblique) [inline node/src/network.rs:61]: Remove `hex` from `Cargo.toml`

File: cli/src/server.rs [hunk 2]
```diff
@@ -65,7 +63,6 @@ pub(crate) async fn run(args: Params) -> Result<()> {
     let state = WasmNodeArgs {
         network: args.network,
         bootnodes,
-        genesis_hash,
```
Comment (oblique) [inline cli/src/server.rs:68]: Remove it from the UI too

File: node/src/network.rs (hunk 1 above)
Comment (fl0rek) [reply node/src/network.rs:61]: And yet unused-deps CI is green 😿 
Removed 👍

File: cli/src/server.rs (hunk 2 above)
Comment (fl0rek) [reply cli/src/server.rs:68]: Fixed our JSs and HTMLs

**Review by zvolin — APPROVED** [review]

**Review by oblique — APPROVED** [review]

**Pull request merged**